domains/kamyzz.json

```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "thanhcong2k7",
+        "email": "nhocpeacock@gmail.com"
+      },
+      "record": {
+        "CNAME": "kamyzz.carrd.me"
```

Comment:
This domain is not registered

Reply:
oh i forgot it was "kmdev.carrd.co" instead of "kamyzz.carrd.me"